plugins/modules/ece_cluster_info.py

```diff
@@ -44,14 +44,29 @@ def main():
         #('alert-type', 'metrics_threshold', ('conditions'))
     
     module = AnsibleModule(argument_spec=module_args, required_if=argument_dependencies, supports_check_mode=True)
+    
+    deployment_name = module.params.get('deployment_name')
+    
     ElasticDeployments = ECE(module)
-    deployment_id = ElasticDeployments.get_deployment_id(module.params.get('deployment_name'))
-    results['deployment_id'] = deployment_id
-   
+    deployment_kibana_info = ElasticDeployments.get_deployment_kibana_info(deployment_name)
+    #results['deployment_kibana_info'] = deployment_kibana_info
+
+    if not deployment_kibana_info:
+      results['deployment_kibana_info'] = "No deployment kibana was returned, check your deployment name"

```

Comment:
We need a standard for the results here. If we are using `deployment_kibana_info` when the deployment isn't found, then we should include that key when it is found. I would say the same for `deployment_kibana_endpoint` and `deployment_kibana_url`. We should give the same keys as part of the results and provide `None` as the value if it should be empty.